plugins/syncbeaconfollower/plugin.go

```diff
@@ -0,0 +1,194 @@
+package syncbeaconfollower
+
+import (
+	"errors"
+	"github.com/iotaledger/goshimmer/packages/binary/messagelayer/message"
+	"github.com/iotaledger/goshimmer/packages/binary/messagelayer/tangle"
+	"github.com/iotaledger/goshimmer/packages/shutdown"
+	"github.com/iotaledger/goshimmer/plugins/config"
+	"github.com/iotaledger/goshimmer/plugins/messagelayer"
+	"github.com/iotaledger/goshimmer/plugins/sync"
+	"github.com/iotaledger/goshimmer/plugins/syncbeacon"
+	"github.com/iotaledger/hive.go/crypto/ed25519"
+	"github.com/iotaledger/hive.go/daemon"
+	"github.com/iotaledger/hive.go/events"
+	"github.com/iotaledger/hive.go/logger"
+	"github.com/iotaledger/hive.go/node"
+	flag "github.com/spf13/pflag"
+	goSync "sync"
+	"time"
+)
+
+const (
+	// PluginName is the plugin name of the sync beacon plugin.
+	PluginName = "Sync Beacon Follower"
+
+	// CfgSyncBeaconFollowNodes defines the list of nodes this node should follow to determine its sync status.
+	CfgSyncBeaconFollowNodes = "syncbeaconfollower.followNodes"
+
+	// CfgSyncBeaconMaxTimeWindowSec defines the maximum time window for which a sync payload would be considerable.
+	CfgSyncBeaconMaxTimeWindowSec = "syncbeaconfollower.maxTimeWindowSec"
+
+	// CfgSyncBeaconMaxTimeOfflineSec defines the maximum time a beacon node can stay without receiving updates.
+	CfgSyncBeaconMaxTimeOfflineSec = "syncbeaconfollower.maxTimeOffline"
+
+	// CfgSyncBeaconCleanupInterval defines the interval that old beacon status are cleaned up.
+	CfgSyncBeaconCleanupInterval = "syncbeaconfollower.cleanupInterval"
+
+	// The percentage of following nodes that have to be synced.
+	syncPercentage = 0.6
+)
+
+// beaconSync represents the beacon payload and the sync status.
+type beaconSync struct {
+	payload *syncbeacon.Payload
+	synced  bool
+}
+
+func init() {
+	flag.StringSlice(CfgSyncBeaconFollowNodes, []string{"Gm7W191NDnqyF7KJycZqK7V6ENLwqxTwoKQN4SmpkB24", "9DB3j9cWYSuEEtkvanrzqkzCQMdH1FGv3TawJdVbDxkd"}, "list of trusted nodes to follow their sync status")
+	flag.Int(CfgSyncBeaconMaxTimeWindowSec, 10, "the maximum time window for which a sync payload would be considerable")
+	flag.Int(CfgSyncBeaconMaxTimeOfflineSec, 40, "the maximum time the node should stay without receiving updates")
```

Comment:
not sure but meaning this?
```suggestion
	flag.Int(CfgSyncBeaconMaxTimeOfflineSec, 40, "the maximum time the node should stay synced without receiving updates")
```